diff --git a/easy-fancybox/inc/admin.js b/easy-fancybox/inc/admin.js
index 253457e8..3ad223a5 100644
--- a/easy-fancybox/inc/admin.js
+++ b/easy-fancybox/inc/admin.js
@@ -92,27 +92,3 @@ wp.domReady( function () {
 	}));
 } );
 
-(function($) {
-	$( document ).ready( function() {
-		var container = $('.efb-review-notice');
-		if ( container.length ) {
-			container.find( '.efb-review-actions a' ).click(function() {
-				container.remove();
-				var rateAction = $( this ).attr( 'data-rate-action' );
-				$.post(
-					ajaxurl,
-					{
-						action: 'efb-review-action',
-						rate_action: rateAction,
-						_n: container.find( 'ul:first' ).attr( 'data-nonce' )
-					},
-					function( result ) {}
-				);
-
-				if ( 'do-rate' !== rateAction ) {
-					return false;
-				}
-			});
-		}
-	});
-})( jQuery );
diff --git a/easy-fancybox/inc/class-easyfancybox-admin.php b/easy-fancybox/inc/class-easyfancybox-admin.php
index 545554c5..0a899275 100644
--- a/easy-fancybox/inc/class-easyfancybox-admin.php
+++ b/easy-fancybox/inc/class-easyfancybox-admin.php
@@ -98,6 +98,7 @@ class easyFancyBox_Admin {
 	 * Show request for plugin review on options page
 	 */
 	public static function show_review_request() {
+		return;
 		// Don't show if not on options screen or dashboard, or if already rated
 		$screen = get_current_screen();
 		$is_dashboard_or_efb_options = 'dashboard' === $screen->id || self::$screen_id === $screen->id;
