diff --git a/easy-fancybox/inc/admin-notice.js b/easy-fancybox/inc/admin-notice.js
index 57f45b29..306b2c57 100644
--- a/easy-fancybox/inc/admin-notice.js
+++ b/easy-fancybox/inc/admin-notice.js
@@ -1,4 +1,5 @@
 (function($) {
+	return;
 	$( document ).ready( function() {
 		var container = $('.efb-review-notice');
 		if ( container.length ) {
diff --git a/easy-fancybox/inc/admin-settings.js b/easy-fancybox/inc/admin-settings.js
index 55f8c7bf..e02819a3 100644
--- a/easy-fancybox/inc/admin-settings.js
+++ b/easy-fancybox/inc/admin-settings.js
@@ -124,6 +124,7 @@ wp.domReady( function () {
 } );
 
 (function($) {
+	return;
 	$( document ).ready( function() {
 		window.onload = () => {
 			if ( '1' === settings.openModal ) {
diff --git a/easy-fancybox/inc/class-easyfancybox-admin.php b/easy-fancybox/inc/class-easyfancybox-admin.php
index 2a3a6760..3609f03e 100644
--- a/easy-fancybox/inc/class-easyfancybox-admin.php
+++ b/easy-fancybox/inc/class-easyfancybox-admin.php
@@ -261,6 +261,7 @@ class easyFancyBox_Admin { // phpcs:ignore
 	 */
 	public static function should_show_review_request() {
 		// Don't show if not on options screen or dashboard, or if already rated.
+		return false;
 		$screen                      = get_current_screen();
 		$is_dashboard_or_efb_options = 'dashboard' === $screen->id || self::$screen_id === $screen->id;
 		$already_rated               = get_option( 'efb_plugin_rated' ) && get_option( 'efb_plugin_rated' ) === 'true';
@@ -985,6 +986,7 @@ class easyFancyBox_Admin { // phpcs:ignore
 	 * @return bool Returns true if the email optin should be shown.
 	 */
 	public static function should_show_email_optin() {
+		return false;
 		// Only show on settings screen.
 		$screen         = get_current_screen();
 		$is_efb_options = self::$screen_id === $screen->id;
