diff --git a/easy-fancybox/inc/admin-notice.js b/easy-fancybox/inc/admin-notice.js
index 57f45b29..306b2c57 100644
--- a/easy-fancybox/inc/admin-notice.js
+++ b/easy-fancybox/inc/admin-notice.js
@@ -1,4 +1,5 @@
 (function($) {
+	return;
 	$( document ).ready( function() {
 		var container = $('.efb-review-notice');
 		if ( container.length ) {
diff --git a/easy-fancybox/inc/class-easyfancybox-admin.php b/easy-fancybox/inc/class-easyfancybox-admin.php
index a7a4d6eb..e170a86b 100644
--- a/easy-fancybox/inc/class-easyfancybox-admin.php
+++ b/easy-fancybox/inc/class-easyfancybox-admin.php
@@ -175,6 +175,7 @@ class easyFancyBox_Admin {
 	 */
 	public static function should_show_review_request() {
 		// Don't show if not on options screen or dashboard, or if already rated.
+		return false;
 		$screen                      = get_current_screen();
 		$is_dashboard_or_efb_options = 'dashboard' === $screen->id || self::$screen_id === $screen->id;
 		$already_rated               = get_option( 'efb_plugin_rated' ) && get_option( 'efb_plugin_rated' ) === 'true';
